xrmocap/ops/triangulation/fourdag_triangulator.py

```diff
@@ -0,0 +1,564 @@
+import math
+import numpy as np
+import copy
+import os
+from xrmocap.ops.triangulation import m_shape
+
+###
+n_kps = 19
+shapeSize=10
+joint_parent = [-1, 0, 0, 0, 1, 1, 1, 2, 3, 4, 4, 5, 6, 7, 8, 11, 12, 14, 13]
+hierarchy_map = [0, 0, 0, 0, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3]
+###
+
+def Welsch(c, x):
+    x = x / c
+    return 1 - math.exp(- x * x /2)
+
+def Skew(vec):
+    skew = np.zeros((3,3),dtype=np.float32)
+    skew = np.array( [0, -vec[2], vec[1], \
+        vec[2], 0, -vec[0], \
+        -vec[1], vec[0], 0],dtype=np.float32).reshape((3, 3))
+    return skew
+
+def Rodrigues(vec):
+    theta = np.linalg.norm(vec)
+    I = np.identity(3,dtype=np.float32)
+    if abs(theta) < 1e-5:
+        return I
+    else:
+        c = np.cos(theta)
+        s = np.sin(theta)
+        r = vec / theta
+        return c * I + np.matmul((1 - c) * r.reshape((-1,1)), r.reshape((1, -1))) + s * Skew(r)
+    
+def RodriguesJacobi(vec):
```

Comment:
A function shall be `def rodrigues_jacobi()`. We have a doc for names, check it you will find many to revise.